ports/zephyr/zephyr_storage.c

```diff
@@ -139,6 +139,20 @@ MP_DEFINE_CONST_OBJ_TYPE(
 #endif // CONFIG_DISK_ACCESS
 
 #ifdef CONFIG_FLASH_MAP
+
+#define FLASH_AREA_DEFINE_LABEL(part) CONCAT(MP_QSTR_, DT_STRING_TOKEN(part, label))
```

Comment:
I know the existing `STORAGE` lookup doesn't have this, but it might be good to add either a prefix or a suffix so the programmer can see this is an ID rather than the partition object itself (that's right, isn't it? I haven't tested this myself.)

Reply:
done